README.md

```diff
@@ -137,8 +247,9 @@ It will also display warnings (but not fail CI)
   may indicate a typo in the path.
 
 Note that some of this validation code is in the
-lib/govuk_web_banners/validators path, which should be tested to ensure the
-checking is valid, but will not be bundled into the released gem.
+lib/govuk_web_banners/validator/recruitment_banner.rb file, which should be
```

Comment:
Maybe worth wrapping this file path in `` for readability?

Reply:
Done